jep-0000/0000/README.adoc

```diff
@@ -0,0 +1,170 @@
+//
+//
+// Uncomment if there will be a BDFL delegate for this JEP.
+| BDFL-Delegate
+| Undetermined
+//
+//
+// Uncomment if discussion will occur in forum other than jenkinsci-dev@ mailing list.
+//| Discussions-To
+//| :bulb: Link to where discussion and final status announcement will occur :bulb:
+//
+//
+// Uncomment if this JEP depends on one or more other JEPs.
+//| Requires
+//| :bulb: JEP-NUMBER, JEP-NUMBER... :bulb:
+//
+//
+// Uncomment and fill if this JEP is rendered obsolete by a later JEP
+//| Superseded-By
+//| :bulb: JEP-NUMBER :bulb:
+//
+//
+// Uncomment when this JEP status is set to Accepted, Rejected or Withdrawn.
+//| Resolution
+//| :bulb: Link to relevant post in the jenkinsci-dev@ mailing list archives :bulb:
+
+|===
+
+
+== Abstract
+
+The Jenkins project wants to allow its users to refer to security vulnerabilities in Jenkins and Jenkins plugins using CVE identifiers.
+To allow for timely assignment of CVE identifiers for vulnerabilities, the Jenkins project needs to become a CVE Numbering Authority, or CNA.
+
+== Specification
+
+The Jenkins project will apply to become a "Sub CNA" CVE numbering authority under the Distributed Weakness Filing (DWF) federated CNA as described in
+link:https://github.com/distributedweaknessfiling/DWF-Documentation#becoming-an-open-source-cna-cve-numbering-authority[this GitHub repository of the DWF Project].
+
+Three documents we need to abide by are being provided by the DWF project:
+
+. To be able to submit data to CVE, every individual involved in providing that data (i.e. the authors of the CVE description, not the reporter(s) of the vulnerability, not the developer(s) fixing it, not the author(s) of the Jenkins security advisory) will need to agree to the https://cve.mitre.org/about/termsofuse.html[MITRE CVE Terms of service].
+. As a CNA operating under the DWF project, the Jenkins project will need to agree to the http://contributor-covenant.org/version/1/4/[DWF Contributor Agreement], which is based on Contributor Covenant 1.4.
+. The Jenkins project will need to agree to the http://cve.mitre.org/cve/cna/rules.html[MITRE CNA rules] that outlines the rules and processes the Jenkins project will need to follow and implement. The following sections apply to the Jenkins project:
+.. Chapter 2 outlines the responsibilities of all CNA, which applies to us.
+.. Chapter 4 outlines the CNA candidate process that includes onboarding.
+
+The Distributed Weakness Filing Project splits the CNA responsibilities into two parts:
+
+* The Jenkins Project, as CNA, will own blocks of CVE numbers.
+* Individuals working with the Jenkins Project, the CVE mentors, will assign CVE numbers and write issue descriptions in the required format.
+
+To start this process, an individual associated with the Jenkins project will need to become a CVE mentor.
+Once they are a CVE mentor, the Jenkins project can apply to become a CNA.
```

Comment:
Once they ~~are~~ have a CVE mentor, no ?

Reply:
No. *They* is a gender-neutral reference to the *individual associated with the Jenkins project*.

Reply:
I suppose both are true, but it's supposed to be implied that 1) someone needs to become a CVE mentor and 2) they need to be available to do the mentoring for the Jenkins project.